app/src/main/res/values/styles.xml

```diff
@@ -264,6 +264,36 @@
     <item name="android:textSize">14sp</item>
     <item name="fontFamily">sans-serif-medium</item>
   </style>
+
+  <style name="InputInteractionEditText" parent="Widget.AppCompat.EditText">
+    <item name="android:layout_width">match_parent</item>
+    <item name="android:layout_height">wrap_content</item>
+    <item name="android:background">@drawable/edit_text_background</item>
```

Comment:
nit fix if you want, we can arrange these items in alphabetical order.

Reply:
I kept it in order of `layout/xml` files where `layout_width` and `layout_height` are always on top. and rest all are in alphabetical order.